deployments/docker-compose.yaml

```
@@ -10,9 +10,7 @@ services:
       retries: 10
     networks:
       - perseus
-    env_file:
-      - required: true
-        path: ../.env
+    env_file:  ../.env
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)